mmv1/templates/terraform/constants/netapp_volume_replication.go.erb

```diff
@@ -0,0 +1,51 @@
+// Custom function to wait for mirrorState target states
+func NetAppVolumeReplicationWaitForMirror(d *schema.ResourceData, meta interface{}, targetState string) error {
```

Comment:
I'm still debating if `wait_for_mirror` is a good feature to support in Terraform, as it may take relatively long time and may add extra complicities and I don't think setting timeouts to O(days) would make sense. You definitely know this resource better than I do, do you think if it make sense if we just let users to use Terraform to initiate actions "stop/resume" but don't wait until the replication is done?